pkg/scheduler/eventhandlers.go

```diff
@@ -385,6 +451,16 @@ func AddAllEventHandlers(
 		},
 	)
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.CSINodeInfo) {
+		csiNodeInformer.Informer().AddEventHandler(
+			cache.ResourceEventHandlerFuncs{
+				AddFunc:    sched.addCSINodeToCache,
+				UpdateFunc: sched.updateCSINodeInCache,
+				DeleteFunc: sched.deleteCSINodeFromCache,
+			},
+		)
+	}
+
```

Comment:
Thanks for adding this here, but I am not 100% sure if `CSINode` object should be cached inside `NodInfo` object. I will defer to @ravisantoshgudimetla @bsalamat on this.